pkg/image/api/helper.go

```diff
@@ -553,10 +553,7 @@ func LatestImageTagEvent(stream *ImageStream, imageID string) (string, *TagEvent
 			continue
 		}
 		for i, event := range events.Items {
-			if event.Image != imageID {
-				continue
-			}
-			if latestTagEvent == nil || (latestTagEvent != nil && event.Created.After(latestTagEvent.Created.Time)) {
+			if digestOrImageMatch(event.Image, imageID) && (latestTagEvent == nil || (latestTagEvent != nil && event.Created.After(latestTagEvent.Created.Time))) {
```

Comment:
@pweil- if latestTagEvent is nil, then the right part will panic(), or i'm drunk?